docs/modules/ROOT/pages/spring-cloud-netflix.adoc

```diff
@@ -267,25 +267,31 @@ Do not use the `EurekaClient` in a `@PostConstruct` method or in a `@Scheduled`
 It is initialized in a `SmartLifecycle` (with `phase=0`), so the earliest you can rely on it being available is in another `SmartLifecycle` with a higher phase.
 ====
 
-==== EurekaClient with Jersey
+==== Underlying HTTP clients
+
+`EurekaClient` uses either `RestTemplate`, `WebClient` or `JerseyClient` under the hood. In order to use the `EurekaClient`, you need to have one of the supported HTTP clients on your classpath.
+
+To use `RestTemplate`, add `spring-boot-starter-web` to your dependencies. To use `WebClient`, add `spring-boot-starter-webflux` to your dependencies. If both `RestTemplate` and `WebClient` are on the classpath, when `eureka.client.webclient.enabled` is set to `true`, `WebClient` will be used. Otherwise, `RestTemplate` will be used.
```

Comment:
Remove the comma before "when" and change both instances of "will be" to "is".

The comma isn't wrong (the sentence works either way), but we aim to remove anything we don't strictly need. The change from "will be" to "is" is in keeping with our practice of sticking with the indefinite present tense (sometimes called the simple present tense), which generally lets us write shorter sentences. Again, you're not wrong, but our conventions do it differently.

Just for reference, here is the Spring Style Guide: https://github.com/spring-io/spring-asciidoctor-backends/blob/main/guides/style-guide.adoc

Reply:
Thanks, @Buzzardo.